### DISCUSSION TITLE
How to set up custom subdomain to a project

### DISCUSSION THREAD

**Discussion by erbill** [vercel/community#2007] [Help] [answered]
### Summary

I'd love to use sample.oursite.com as a main domain of our Vercel app, but don't want to change DNS settings or nameservers for the root domain. How can I set up Vercel app through just one subdomain but not effecting the root domain?

Thanks

**Comment by amyegan** [comment, marked as answer]
You can add a subdomain without changing the behavior of your root domain by using a CNAME record. 

First, go to your Vercel Project settings and add the subdomain

![image](https://user-images.githubusercontent.com/8247105/230631843-724cd57f-c15d-47d9-961b-456767b2ce8a.png)

Then, go to your DNS provider and add the CNAME record. Propagation of [CNAME record changes](https://vercel.com/guides/why-is-my-vercel-domain-unverified#cname-record-changes) is usually very quick.

**Reply by thenabeelhassan** [reply]
Hi @amyegan 

I am in similar kind of issue; I have done all steps mentioned above about 36hours ago. I am still facing the Invalid Configuration message on Vercel.

I have also created a issue with some more details. [2348](https://github.com/orgs/vercel/discussions/2348).

**Reply by amyegan** [reply]
Hi @thenabeelhassan. You can check DNS propagation status using a tool like [whatsmydns.net](https://www.whatsmydns.net/) or `dig`

Our documentation and help guides have more info:

[Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)
[DNS propagation](https://vercel.com/docs/concepts/projects/domains#dns-propagation)
[Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)

**Reply by thenabeelhassan** [reply]
@amyegan I have checked the DNS and my domain name servers are pointing the sub domain to Vercel.

**Comment by erbill** [comment]
Thank you that worked flawlessly!
